[2024-09-25T07:45:59.642+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-25T07:45:59.662+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: random_forest_classifier_dag.rfc_save_results manual__2024-09-24T20:36:41.723899+00:00 [queued]>
[2024-09-25T07:45:59.672+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: random_forest_classifier_dag.rfc_save_results manual__2024-09-24T20:36:41.723899+00:00 [queued]>
[2024-09-25T07:45:59.674+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-09-25T07:45:59.689+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): rfc_save_results> on 2024-09-24 20:36:41.723899+00:00
[2024-09-25T07:45:59.702+0000] {standard_task_runner.py:64} INFO - Started process 16116 to run task
[2024-09-25T07:45:59.705+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'random_forest_classifier_dag', 'rfc_save_results', 'manual__2024-09-24T20:36:41.723899+00:00', '--job-id', '590', '--raw', '--subdir', 'DAGS_FOLDER/***/dags/.ipynb_checkpoints/hw1-checkpoint.py', '--cfg-path', '/tmp/tmps0w12ut1']
[2024-09-25T07:45:59.708+0000] {standard_task_runner.py:91} INFO - Job 590: Subtask rfc_save_results
[2024-09-25T07:45:59.765+0000] {task_command.py:426} INFO - Running <TaskInstance: random_forest_classifier_dag.rfc_save_results manual__2024-09-24T20:36:41.723899+00:00 [running]> on host 82997fc6deb7
[2024-09-25T07:45:59.896+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='alexvlg34@gmail.com' AIRFLOW_CTX_DAG_OWNER='Alex Trigolos' AIRFLOW_CTX_DAG_ID='random_forest_classifier_dag' AIRFLOW_CTX_TASK_ID='rfc_save_results' AIRFLOW_CTX_EXECUTION_DATE='2024-09-24T20:36:41.723899+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-24T20:36:41.723899+00:00'
[2024-09-25T07:45:59.898+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-25T07:45:59.914+0000] {hw1-checkpoint.py:167} INFO - Save metrics...
[2024-09-25T07:46:00.078+0000] {base.py:84} INFO - Using connection ID 's3_connection' for task execution.
[2024-09-25T07:46:00.080+0000] {connection_wrapper.py:388} INFO - AWS Connection (conn_id='s3_connection', conn_type='aws') credentials retrieved from login and password.
[2024-09-25T07:46:00.781+0000] {hw1-checkpoint.py:194} INFO - DONE!!!
[2024-09-25T07:46:00.784+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-09-25T07:46:00.784+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-25T07:46:00.799+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=random_forest_classifier_dag, task_id=rfc_save_results, run_id=manual__2024-09-24T20:36:41.723899+00:00, execution_date=20240924T203641, start_date=20240925T074559, end_date=20240925T074600
[2024-09-25T07:46:00.843+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-09-25T07:46:00.867+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-25T07:46:00.871+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
